new-system-tests/src/test/java/io/apicurio/registry/systemtests/SqlNoIAM.java

```diff
@@ -0,0 +1,104 @@
+package io.apicurio.registry.systemtests;
+
+import io.apicur.registry.v1.ApicurioRegistry;
+import io.fabric8.junit.jupiter.api.KubernetesTest;
+import io.fabric8.junit.jupiter.api.LoadKubernetesManifests;
+import io.fabric8.openshift.client.OpenShiftClient;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+import static io.apicurio.registry.systemtests.Utils.findRegistryOperatorDeployment;
+import static io.apicurio.registry.systemtests.Utils.isDeploymentReady;
+import static io.apicurio.registry.systemtests.Utils.waitDeploymentReady;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@KubernetesTest
+@LoadKubernetesManifests({
+        // PostgreSQL database resources
+        "/sql/00_service.yaml", // Service
+        "/sql/01_deployment.yaml", // Deployment
+        // Apicurio Registry operator
+        "/apicurio/00_operator_group.yaml", // Operator group for Apicurio Registry operator
+        "/apicurio/01_subscription.yaml", // Apicurio Registry operator subscription
+        // Apicurio Registry instance
+        "/apicurio/02_registry_sql_no_iam.yaml" // Apicurio Registry instance with PostgreSQL storage and without IAM
+})
+@TestMethodOrder(MethodOrderer.MethodName.class)
```

Comment:
Requiring a concrete execution order is usually a red flag.

Reply:
Ou, you are right, I forgot this line here when testing this "feature". It is removed now.